index.html

```diff
@@ -182,181 +198,34 @@ <h2 id="project_tagline"></h2>
-<pre><code><strong>HTTP Response:</strong> 200 :OK
-[
-	{
-	 "id":"9cdd0f68-df85-44b0-88bd-fc4163ac90a0",
-	 "status_code":200,
-	 "result":"SUCCESS",
-	 "message":"Event sent successfully"
-	},
-	{
-	 "id":"9cdd0f68-df85-44b0-88bd-fc4163ac90a1",
-	 "status_code":200,
-	 "result":"SUCCESS",
-	 "message":"Event sent successfully"
-	}
-]
-</code></pre>
-<p><strong>400:</strong>When the input JSON is malformated, that particular events internal status code will be 400. </p>
-<pre><code><strong>HTTP Response:</strong> 400 :Bad Request
-[
-	{
-	 "status_code":400,
-	 "result":"Bad Request",
-	 "message":"Invalid event content, client need to fix problem in event before submitting again"
-	},
-]
-</code></pre>
-<p><strong>207:</strong>When the events are having different internal status codes,then the status code of HTTP response will be 207.</p>
-<pre><code><strong>HTTP Response:</strong> 207 :Multi Status
-[
-	{
-	 "id":"9cdd0f68-df85-44b0-88bd-fc4163ac90a0",
-	 "status_code":200,
-	 "result":"SUCCESS",
-	 "message":"Event sent successfully"
-	},
-	{
-	 "status_code":400,
-	 "result":"Bad Request",
-	 "message":"Invalid event content, client need to fix problem in event before submitting again"
-	},
-]
-</code></pre>
-<p><strong>500:</strong>We will get this status code,When the event is failed to send because of internal server error.</p>
-<pre><code><strong>HTTP Response:</strong> 500 :Internal Server Error
-[
-	{
-	 "id":"9cdd0f68-df85-44b0-88bd-fc4163ac90a0",
-	 "status_code":500,
-	 "result":"Internal Server Error",
-	 "message":"Possible to try again later when server is up"
-	},
-]
-</code></pre>
-<p><strong>503:</strong>Event will have this status code when the previous event is failed to send.</br>
-For example,Consider you have 5 events.</br>
-E1<-E2<-E3<-E4<-E5</br>
-You send them all to publish in a batch command.</br>
-E1 and E2 is successful. (200)</br>
-E3 is malformatted. (400)</br>
-If we send E4 and E5 we will have a broken chain so that is not a good option. In this case we should skip sending E4 and E5 and have status code 503 for those.</p>
-<pre><code><Strong>HTTP Response</strong>: 207 :Multi Status
-[	
-	{
-	 "id":"9cdd0f68-df85-44b0-88bd-fc4163ac90a0",
-	 "status_code":200,
-	 "result":"SUCCESS",
-	 "message":"Event sent successfully"
-	},
-	{
-	 "id":"9cdd0f68-df85-44b0-88bd-fc4163ac90a1",
-	 "status_code":200,
-	 "result":"SUCCESS",
-	 "message":"Event sent successfully"
-	},
-	{
-	 "status_code":400,
-	 "result":"Bad Request",
-	 "message":"Invalid event content, client need to fix problem in event before submitting again"
-	},
-	{
-	 "status_code":503,
-	 "result":"Service Unavailable",
-	 "message":"Please check previous event and try again later"
-	},
-	{
-	 "status_code":503,
-	 "result":"Service Unavailable",
-	 "message":"Please check previous event and try again later"
-	},
-]</code></pre>
 <p>You can open the RabbitMQ's management console and find these messages in a queue.</p>
```

Comment:
Why is this removed? Doesn't this reflect the changes made by us for adding StatusCodes into the document?